torso/myotorso_exosuit.xml

```diff
@@ -9,6 +9,7 @@
 
   <include file="../scene/myosuite_scene.xml"/>
   <include file="assets/myotorso_assets.xml"/>
+  <include file="../../myo_sim/head/assets/myohead_simple_assets.xml"/>
```

Comment:
include relative paths are all resolved differently. This will cause problems later